governance/votes/Executive vote - May 10, 2023.md

```diff
@@ -139,18 +144,36 @@ To facilitate the MKR vesting stream to Phoenix Labs, the `cap` in the MKR_VEST
 
 This will allow for the full agreed amount of MKR to be streamed to PullUp along with a 10% buffer.
 
+### DAO Resolution
+
+As per [this post](https://forum.makerdao.com/t/dao-resolution-to-facilitate-onboarding-of-taco-with-exchange-agent/20735) from the Strategic Finance Core Unit (SF-001), the following DAO Resolution will be approved if this executive proposal passes. 
```

Comment:
Should this refer to steakhouse instead? I recall we referred to dewiz versus CES elsewhere.